cpu/native/syscalls.c

```diff
@@ -237,7 +246,18 @@ ssize_t _native_write(int fd, const void *buf, size_t count)
 
 ssize_t _native_writev(int fd, const struct iovec *iov, int iovcnt)
 {
-    ssize_t r;
+    ssize_t r = 0;
+
+    if (fd == STDOUT_FILENO || fd == STDERR_FILENO) {
+        for (int i = 0; i < iovcnt; ++i) {
+            ssize_t res = stdio_write(iov->iov_base, iov->iov_len);
+            iov++;
```

Comment:
I think `res` needs to be compared to `iov->iov_len` before advancing to next element,
(there are stdio implementations in RIOT that might not write everything given to them (avoid blocking))

Reply:
240232668605301d13273be97780090ade9c6241 should fix this

(Is this function actually ever used in RIOT?)

Reply:
> (Is this function actually ever used in RIOT?)

afaik there is/was something in glibc that uses it for something more common